components/user_settings/display/manage_timezones.jsx

```diff
@@ -0,0 +1,245 @@
+            </div>
+        );
+
+        const noTimezonesFromServer = timezones.length === 0;
+        const automaticTimezoneInput = (
+            <div className='checkbox'>
+                <label>
+                    <input
+                        id='automaticTimezoneInput'
+                        type='checkbox'
+                        checked={useAutomaticTimezone}
+                        onChange={this.handleAutomaticTimezone}
+                        disabled={noTimezonesFromServer}
+                    />
+                    <FormattedMessage
+                        id='user.settings.timezones.automatic'
+                        defaultMessage='Set automatically'
+                    />
+                    {timezoneRegion}
+                </label>
+            </div>
+        );
+
+        const providers = [new TimezoneProvider()];
+        const manualTimezoneInput = (
+            <div key='changeTimezone'>
+                <br/>
+                <label className='control-label'>
+                    <FormattedMessage
+                        id='user.settings.timezones.change'
+                        defaultMessage='Change timezone'
+                    />
+                </label>
+                <div className='padding-top'>
+                    <SuggestionBox
+                        ref={this.setSwitchBoxRef}
+                        className='form-control focused'
+                        type='search'
+                        onChange={this.onChange}
+                        value={this.state.manualTimezoneInput}
+                        onItemSelected={this.handleTimezoneSelected}
+                        listComponent={SuggestionList}
+                        maxLength='64'
+                        requiredCharacters={0}
+                        providers={providers}
+                        listStyle='bottom'
+                        completeOnTab={false}
+                        renderDividers={false}
+                        openOnFocus={true}
+                        disabled={noTimezonesFromServer}
+                    />
+                    {serverError}
+                </div>
+                <div>
+                    <br/>
+                    <FormattedHTMLMessage
+                        id='user.settings.timezones.promote'
+                        defaultMessage='Select the time zone used for timestamps in the user interface and email notifications.'
+                    />
+                </div>
+            </div>
+        );
+
+        inputs.push(automaticTimezoneInput);
+
+        if (!useAutomaticTimezone) {
+            inputs.push(manualTimezoneInput);
+        }
+
+        return (
+            <SettingItemMax
+                title={
+                    <FormattedMessage
+                        id='user.settings.display.timezone'
+                        defaultMessage='Timezone'
+                    />
+                }
+                containerStyle={{
```

Comment:
Instead of passing in a raw style object, could you move this into the CSS and change this to a className that you're passing in?